docs/pages/database-access/reference/cli.mdx

```diff
@@ -150,6 +150,26 @@ $ teleport db configure bootstrap -c /etc/teleport.yaml --manual
 | `--attach-to-role` | Role name to attach the policy to. Mutually exclusive with `--attach-to-user`. If none of the attach-to flags is provided, the command will try to attach the policy to the current user/role based on the credentials. |
 | `--attach-to-user` | User name to attach the policy to. Mutually exclusive with `--attach-to-role`. If none of the attach-to flags is provided, the command will try to attach the policy to the current user/role based on the credentials. |
 
+## teleport db configure aws print-iam
+
+Print the necessary IAM permissions required for the Database Service based on
+provided database types.
+
+```code
+$ teleport db configure aws print-iam --types rds
+$ teleport db configure aws print-iam --types rds,redshift --role my-db-service-role
+$ teleport db configure aws print-iam --types redshift-serverless --assumes-roles my-access-role --policy
+```
+
+| Flag | Description |
+| - | - |
+| `-r/--types` | Comma-separated list of database types to include in the policy. Any of rds,rdsproxy,redshift,redshift-serverless,elasticache,memorydb,keyspace,dynamodb,opensearch. |
```

Comment:
I would use code-style for each possible value. Otherwise, I found the different possible values difficult to parse:

<img width="752" alt="image" src="https://github.com/gravitational/teleport/assets/25250668/aa053d62-ab56-4e62-916b-5422bd4257da">